source/constructs/api/data_source/crud.py

```diff
@@ -283,6 +322,32 @@ def update_jdbc_instance_count(account: str, region: str):
     session.merge(account)
     session.commit()
 
+def create_jdbc_connection(provider_id: int,
+                           account: str,
+                           region: str,
+                           instance: str,
+                           glue_connection: str,
+                           glue_database: str,
+                           glue_vpc_endpoint_id: str,
+                           crawler_name: str):
+    session = get_session()
+    jdbc_connection_source = session.query(JDBCInstanceSource).filter(JDBCInstanceSource.account_provider_id == provider_id,
+                                                                      JDBCInstanceSource.account_id == account,
+                                                                      JDBCInstanceSource.region == region,
+                                                                      JDBCInstanceSource.instance_id == instance).order_by(
+        desc(JDBCInstanceSource.detection_history_id)).first()
+    if jdbc_connection_source is None:
+        jdbc_connection_source = JDBCInstanceSource(instance_id=instance, region=region, aws_account=account)
+    jdbc_connection_source.glue_database = glue_database
+    jdbc_connection_source.glue_crawler = crawler_name
+    jdbc_connection_source.glue_connection = glue_connection
+    jdbc_connection_source.glue_vpc_endpoint = glue_vpc_endpoint_id
+    jdbc_connection_source.glue_crawler_last_updated = datetime.datetime.utcnow()
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Naive datetime objects are treated by many datetime methods as local times, We recommend that you create aware timestamps with the `now` or `fromtimestamp` functions using the `tzinfo` to attribute to represent times in UTC. Do not use the [datetime.utcnow()](https://docs.python.org/3/library/datetime.html#datetime.datetime.utcnow) because it returns only a `naive` datetime object. 

[Learn more](https://docs.python.org/3/library/datetime.html#aware-and-naive-objects)